twitter_scraper.py

```diff
@@ -54,7 +54,7 @@ def gen_tweets(pages):
                     for style in styles:
                         if style.startswith('background'):
                             tmp = style.split('/')[-1]
-                            video_id = tmp[:tmp.index('.jpg')]
+                            video_id = '.'.join(tmp.split('.')[:-1])
```

Comment:
Looks like it fixes the crash from this line.  Thoughts @ParthS007 @bisguzar?

Reply:
It's scraping thumbsnail picture, and it has been created by Twitter. So I think JPG is default for it. We need more information about problem to fix it.

Here is an example, look like its scraping IDs correctly.
```python
>>> from twitter_scraper import get_tweets                                                                                                                                                                           

>>> for tweet in get_tweets('TwitterVideo', 1): 
...     print(tweet['entries']['videos'])                                                                                                                                                                            
[]
[{'id': 'EHXi82YWoAAURf6'}]
[{'id': 'PkaZjchGOqS52OpA'}]
[{'id': '4qTCo73r2CeEw6Td'}]
[{'id': 'wY3Hl8GgRJxLhyM4'}]
[]
[]
[{'id': 'hW-cu3Rf6oD5o8AE'}]
[{'id': 'GWR1wLiw1-RljBpl'}]
[{'id': 'EFzMdRHW4AA7as1'}]
[{'id': 'BougBSvz7VvvgHlQ'}]
[]
[{'id': 'pNWyUExsfkFpPRo9'}]
[]
[{'id': 'cmsD6W9JRnhOTWg4'}]
[]
[{'id': 'nCK_pxrk9MLA4fLA'}]
[]
[]
[{'id': '4qprWG5Shg6QsQrm'}]
[]
```

It's very old PR, @BenQuigley  what was your motivation for this PR?

Reply:
@bisguzar I think @BenQuigley and I saw the same crash. The line causes the following error:

### error:
```
Traceback (most recent call last):
  File "/Users/george/projects/outreach/outreach-backend/crawler/twitter.py", line 11, in <module>
    for tweet in tweets:
  File "/Users/george/.local/share/virtualenvs/outreach-backend-qJG-ryx7/lib/python3.7/site-packages/twitter_scraper.py", line 78, in get_tweets
    yield from gen_tweets(pages)
  File "/Users/george/.local/share/virtualenvs/outreach-backend-qJG-ryx7/lib/python3.7/site-packages/twitter_scraper.py", line 57, in gen_tweets
    video_id = tmp[:tmp.index('.jpg')]
ValueError: substring not found
```

### details:
Here's the retweet that seems to be causing it: https://twitter.com/wendys/status/816481993944891392

The `style` variable when it crashes is `'background-image:url(\'https://pbs.twimg.com/card_img/1193354949997940737/eLTNI2Y9?format=jpg&name=280x280\')'`.  This seems to be a thumbnail of an embedded video

### reproduce 

Here's the code (the tweet is on page 40 something):

```python
from twitter_scraper import get_tweets
tweets = get_tweets('wendys', pages=50)
for tweet in tweets:
    tweetId = tweet['tweetId']
    if tweetId == '816481993944891392':
        print(tweet)
```

I'm not sure if this PR is the entire solution, because using @BenQuigley 's code causes another error `html = HTML(html=r.json()['items_html'],`

Thanks for the great work! 😄